### PR TITLE
Adding README information tailored to Windows contributors 

### PR DESCRIPTION
**Pull request by bmoe24x**
Was a couple things in the docs that I thought could make the contribution process a bit easier for Windows users

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,11 +81,19 @@ cd CrewLink
...
+Mac/Linux:
 ```sh
 mkdir -p node_modules\iohook\builds\electron-v80-win32-x64\build\Release\
 cp iohook\iohook.node node_modules\iohook\builds\electron-v80-win32-x64\build\Release\
 cp iohook\uiohook.dll node_modules\iohook\builds\electron-v80-win32-x64\build\Release\
 ```
+Windows:
+```sh
+mkdir -p node_modules/iohook/builds/electron-v80-win32-x64/build/Release/
+cp iohook/iohook.node node_modules/iohook/builds/electron-v80-win32-x64/build/Release/
```
Comment (stevebell117) [inline README.md:94]: cp is not a valid command if you don't use `git-bash`. 

Window's native is `copy` instead for command line (or terminal, for those who use VS/VSCode)

Comment (sjoerd108) [reply README.md:94]: On top of this, the -p parameter does not apply to the windows version of the command. It'll instead make a folder labelled "-p"

Comment (bmoe24x) [reply README.md:94]: you both are very much correct and as you hinted I was indeed using bash in VS Code

Comment (bmoe24x) [reply README.md:94]: @sjoerd108 @stevebell117 do yall know what the original commands are actually for? Doesn't mac/linux use `/` for creating directories? I was just trying to be helpful because when using `bash` and Windows there was an extra Visual Studio dependencies and the original commands had strange behavior

Comment (sjoerd108) [reply README.md]: I'm not quite sure what tools let you use Unix based utilities with windows path separators.

Comment (bmoe24x) [reply README.md]: I was wondering the same thing

**Comment by cinderblock** [comment]
#263 makes most of this irrelevant.

PS, I'd personally recommend VS Code over Visual Studio ;)

**Comment by bmoe24x** [comment]
> #263 makes most of this irrelevant.
> 
> PS, I'd personally recommend VS Code over Visual Studio ;)

Didnt see that, makes sense. And I never said I was using Studio, `yarn install` errored without Studio dependencies.

File: README.md [hunk 2]
```diff
@@ -63,6 +63,7 @@ Server code is located at [ottomated/CrewLink-server](https://github.com/ottomat
 This is an example of how to list things you need to use the software and how to install them.
 * [Python](https://www.python.org/downloads/)
 * [node.js](https://nodejs.org/en/download/)
+* [Visual Studio](https://visualstudio.microsoft.com/downloads/) (Windows)
```
Comment (cinderblock) [inline README.md:66]: @bmoe24x here is where you reference Visual Studio. Sure, Visual Studio Code is on there, but it's below the fold for me.

Comment (bmoe24x) [reply README.md:66]: I am aware that I added it as dependency, but not for the IDE capability. `yarn` was complaining it needed Visual Studio as a dependency

Comment (cinderblock) [reply README.md:66]: What do you mean by "dependency"? I don't see any changes to the dependencies in `package.json`. Do you just mean how it's under the "Prerequisites" section? I guess yeah, technically VS Code is not required to edit this. People can use whatever editor they want.

Comment (stevebell117) [reply README.md:66]: @cinderblock If you try to run `yarn dev` on a Windows machine, it fails with the following error:

```
gyp ERR! find VS
gyp ERR! find VS **************************************************************
gyp ERR! find VS You need to install the latest version of Visual Studio
gyp ERR! find VS including the "Desktop development with C++" workload.
gyp ERR! find VS For more information consult the documentation at:
gyp ERR! find VS https://github.com/nodejs/node-gyp#on-windows
gyp ERR! find VS **************************************************************
gyp ERR! find VS
```

This is why he had added the `Visual Studio` dependency, although admittedly it wasn't fully explained.

Comment (cinderblock) [reply README.md:66]: Oh! That's a different issue.

The official line from Microsoft on how to setup your machine seems to be here: https://github.com/microsoft/nodejs-guidelines/blob/master/windows-environment.md#prerequisites

TL;DR, use [windows-build-tools](https://www.npmjs.com/package/windows-build-tools): `npm install --global windows-build-tools`

Alternatively, the manual way you described works.
